javaScriptLoops-forLoop/practiceProblems/sampleBreak.js

```diff
@@ -0,0 +1,12 @@
+let message = "supercalifragilisticexpialidocious";
+message = message.length;
```

Comment:
I just realized the indirection here. I think we can rewrite this into:
`const characterCount = "supercalifragilisticexpialidocious".length;`

This way it's clearer what the variable's value is by renaming it, and we did not need to reassign the variable's value.

Reply:
Noted on the improvement. PR updated for review @lycoangelo 

https://github.com/nicoRDL/dotTheory-JS/pull/12/commits/b08d8d041ea54a75bb78853adef1a5d5e9d73207